--- /usr/local/lib/python3.6/dist-packages/torch/nn/modules/loss.py
+++ /usr/local/lib/python3.6/dist-packages/torch/nn/modules/loss.py
@@ -21,7 +21,7 @@
     :math:`x` and :math:`y` are tensors of arbitrary shapes with a total
     of :math:`n` elements each.
 
-    The mean operation still operates over all the elements, and divides by :math:`n`.
+    The sum operation still operates over all the elements, and divides by :math:`n`.
 
     The division by :math:`n` can be avoided if one sets ``reduction = 'sum'``.
 